test/swarming/bot-scripts/botutil.py

```diff
@@ -32,6 +32,10 @@ def runcmd(cmd):
     '''Run a command, redirecting output to the system stdout and stderr.'''
     return subprocess.run(cmd, stdout=sys.stdout, stderr=sys.stderr)
 
+def runcapture(cmd):
+    '''Run a command, capturing output.'''
+    return subprocess.run(cmd, capture_output=True, text=True)
```

Comment:
I would prefer to rename to `runSaveOutput`, because "capture" already means something on AGI lingo.

Reply:
I think I will just modify runcmd() with a default named parameter save_output set to false